docs/CHANGELOG.rst

```diff
@@ -20,6 +20,7 @@ Added
   Docker images
 - Add ``resolwebio/xengsort:1.0.0`` Docker image
 - Create new Docker image ``resolwebio/qc:1.0.0`` and add RNA-SeQC tool (version 2.4.2)
+- Add ``collapse_annotation.py`` script to qc image
```

Comment:
Be explicit when referencing the Docker image version. It's easier to track changes in the future. In this case the changelog should be:

Add ``collapse_annotation.py`` script to ``resolwebio/qc:1.1.0`` image